src/praisonai/praisonai/cli/commands/config.py

```
@@ -99,12 +122,43 @@ def config_set(
+    else:  # project
+        config_path = Path.cwd() / ".praisonai" / "config.yaml"
+        config_path.parent.mkdir(parents=True, exist_ok=True, mode=0o755)
+    
+    # Load existing config
+    config = {}
+    if config_path.exists():
+        try:
+            with open(config_path, 'r') as f:
+                config = yaml.safe_load(f) or {}
+        except Exception:
+            pass
```

Ruff (0.15.17)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)